app/views/simple_smart_answers/_node.html.erb

```diff
@@ -33,7 +35,7 @@
           <%= o.link_to_remove "<i class=\"glyphicon glyphicon-remove glyphicon-smaller-than-text add-right-margin\"></i> Remove option".html_safe,
             :class => "btn btn-default add-left-margin remove-option" %>
           <% if o.object.errors.has_key?(:next_node) -%>
-            <span class="help-inline"><%= o.object.errors[:next_node].join(', ') %></span>
+            <span class="has-error help-inline"><%= o.object.errors[:next_node].join(', ') %></span>
```

Comment:
Does this render properly? it looked a bit funky when I tried.

(as an aside we probably should have had this bit as a separate PR too)

Reply:
No, it doesn't. To be fair this isn't something we've introduced - it is broken in `main` and the `has-error` class has no other effect than turning the `can't be blank` message red. I will raise another PR and fix the issue there so as not to block this one any further.